gateway/1.4/modules/ROOT/pages/flex-gateway-securing-conn.adoc

```diff
@@ -0,0 +1,13 @@
+= Securing and Governing APIs in Connected Mode
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+For Flex Gateway running in Connected Mode, you apply API policies using *API Manager*. Refer to xref:policies-included-apply.adoc[] for more information.
+
+Additionally, xref:flex-gateway-getting-started.adoc[] describes how to apply a basic authentication policy in Connected Mode.
```

Comment:
Suggest flipping:
To apply a basic...., see xref

Reply:
Fixed